scripts/000_install_base.sh

```diff
@@ -10,6 +10,9 @@ pushd $(dirname $0)
 ./040_install_go.sh
 ./050_install_python.sh
 ./050_install_supervisor.sh
+./051_install_mongodb.sh
```

Comment:
We dont need these in base. You can add them to the json file for the specific purpose.

Reply:
They are needed for development purposes. That's why I thought it'd be better to have them in the base considering container images to be used for both development and deployment.